src/MentionWrapper.js

```diff
@@ -43,20 +43,23 @@ class MentionWrapper extends Component {
     const tokens = textBeforeCaret.split(/\s/);
     const lastToken = tokens[tokens.length - 1];
 
-    // check if the text befor the caret ends with the last word
-    const triggerIdx = textBeforeCaret.endsWith(lastToken)
-      ? textBeforeCaret.length - lastToken.length
-      : -1;
+    // Get active trigger
+    const activeTrigger = this.triggers.find(trigger => lastToken.includes(trigger)) || 0;
```

Comment:
Triggers are strings right? `activeTrigger.length` below will break if you set it to `0`

Reply:
Interesting...it all still works since `subStr` doesn't throw a fit, but you are right :)